content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md

```diff
@@ -0,0 +1,231 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2-tba (June 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+compatibleOSSVersion: Redis 7.2.0
+description: region_name field for new AWS S3 regions. Bug fixes.
+linkTitle: 7.4.2-tba (June 2024)
+weight: 67
+---
+
+This is a maintenance release for ​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/).
+
+## Highlights
+
+This version offers:
+
+- `region_name` field for new AWS S3 regions
+
+- Bug fixes
+
+## New in this release
+
+### Enhancements
+
+- Added the `region_name` field in the REST API to import, export, and back up data in new AWS S3 regions.
+
+#### Redis module feature sets
+
+Redis Enterprise comes packaged with several modules. As of version 7.4.2, Redis Enterprise includes two feature sets, compatible with different Redis database versions.
+
+Bundled Redis modules compatible with Redis database version 7.2:
+
+- [RediSearch 2.8.13]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md" >}})
+
+- [RedisJSON 2.6.10]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.6-release-notes.md" >}})
+
+- [RedisTimeSeries 1.10.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.10-release-notes.md" >}})
+
+- [RedisBloom 2.6.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.6-release-notes.md" >}})
+
+- [RedisGears 2.0.20](https://github.com/RedisGears/RedisGears/releases/tag/v2.0.20-m21)
```

Comment:
Please change "RedisGears 2.0.20" to  "RedisGears 2.0.20 (preview)"

Reply:
Fixed